terraform/modules/active_directory_dns_resolver/main.tf

```diff
@@ -25,6 +30,7 @@ locals {
 
 resource "aws_vpc_dhcp_options" "dns_resolver" {
   domain_name_servers = local.dns_servers
+  domain_name         = var.dns_search
```

Comment:
(this is undone in the next PR - the MarkLogic stack seems to need it to be region.compute.internal as you presumably discovered a couple of weeks ago!)

Reply:
Yeah, it's kinda annoying but that's what I found